samples/nrf9160/nrf_cloud_mqtt_multi_service/src/cloud_connection.c

```diff
@@ -438,43 +340,59 @@ static void cloud_event_handler(const struct nrf_cloud_evt *nrf_cloud_evt)
 		LOG_DBG("NRF_CLOUD_EVT_USER_ASSOCIATION_REQUEST");
 		/* This event indicates that the user must associate the device with their
 		 * nRF Cloud account in the nRF Cloud portal.
+		 *
+		 * The device must then disconnect and reconnect to nRF Cloud after association
+		 * succeeds.
 		 */
 		LOG_INF("Please add this device to your cloud account in the nRF Cloud portal.");
 
-		/* Notify that we have been requested to associate with a user account.
+		/* Store the fact that this is an initial association.
+		 *
 		 * This will cause the next NRF_CLOUD_EVT_USER_ASSOCIATED event to
 		 * disconnect and reconnect the device to nRF Cloud, which is required
 		 * when devices are first associated with an nRF Cloud account.
 		 */
-		notify_cloud_requested_association();
+		atomic_set(&initial_association, true);
 		break;
 	case NRF_CLOUD_EVT_USER_ASSOCIATED:
 		LOG_DBG("NRF_CLOUD_EVT_USER_ASSOCIATED");
 		/* Indicates successful association with an nRF Cloud account.
-		 * Will be fired every time the device connects.
-		 * If an association request has been previously received from nRF Cloud,
-		 * this means this is the first association of the device, and we must disconnect
-		 * and reconnect to ensure proper function of the nRF Cloud connection.
+		 * Devices must associate each time they connect.
```

Comment:
`Devices must associate each time they connect.`
This is not true.

Reply:
You're right, we mean something else by association

I was looking for something a bit more direct, though, than "this event will be fired every time the device connects"

I'd like to be able to say, maybe: "The device associates with XXX every time it connects. If this is the first association..." and so on. I'm not sure what the correct terminology would be.

I could just revert to `Will be fired every time the device connects.` but that seemed less clear to me

Reply:
This event indicates that the device is associated with an account.

Reply:
How's the updated phrasing on this?